requirements.txt

```diff
@@ -0,0 +1,2 @@
+PyGithub
+requests
```

Comment:
Seeing this is a application/tool (not a library), let's pin these to specific versions.